client/app/components/packages/applicant-fieldset.hbs

```diff
@@ -0,0 +1,48 @@
+<fieldset class="fieldset relative">
+  <legend>
+    <strong>
+      {{@title}}
+    </strong>
+  </legend>
+
+  <div class="grid-x grid-margin-x">
+    <div class="cell medium-6">
+      <label>First Name <input type="text"></label>
+    </div>
+    <div class="cell medium-6">
+      <label>Last Name <input type="text"></label>
+    </div>
+  </div>
+  
+  <label>Organization <input type="text"></label>
+  <label>Email Address <input type="email"></label>
+  <label>Address <input type="text"></label>
+
+  <div class="grid-x grid-margin-x">
+    <div class="cell medium-auto">
+      <label>City <input type="text"></label>
+    </div>
+    <div class="cell auto medium-2">
+      <label>State <input type="text"></label>
+    </div>
+    <div class="cell auto medium-4">
+      <label>ZIP <input type="text"></label>
+    </div>
+  </div>
+  
+  <label>Phone <input type="text"></label>
+
+  {{!-- Remove button if not Primary Applicant --}}
+  {{#if (not-eq @title "Primary Applicant") }}
```

Comment:
Nit that I waffle on this a bit, but what do you think about having this logic inside the component class as a computed property? Generally if the logic goes beyond "if truthy", I make a computed. Sometimes that doesn't work if you have a loop, but in this case it might make sense.

Reply:
@allthesignals this ended up being amended after learning more about the two applicant tables in CRM, but I see your point and agree that it would be clearer to have anything but the most basic evaluations in a computed rather than doing anything complicated in the template logic.  The way this was written is also imperative and kinda hides the business logic being implemented.